lib/utils/__tests__/FileCache.test.js

```diff
@@ -0,0 +1,119 @@
+"use strict"
+
+const sinon = require("sinon")
+const mock = require("mock-require")
+const fileEntryCache = require("file-entry-cache")
+let FileCache = require("../FileCache")
+
+const hashOfConfig = "hashOfConfig"
+it("cache file is created when empty cacheLocation", () => {
+  // create mock
+  const fileEntryCacheInstance = {}
+  const mockedFileEntryCache = sinon.mock(fileEntryCache)
+  mockedFileEntryCache.expects("create").once().returns(fileEntryCacheInstance)
+  mock("file-entry-cache", mockedFileEntryCache)
+  FileCache = mock.reRequire("../FileCache")
+
+  const fileCache = new FileCache(undefined, hashOfConfig)
+  // assertions
+  expect(fileCache._fileCache).toBe(fileEntryCacheInstance)
+  expect(fileCache._hashOfConfig).toBe(hashOfConfig)
+  mockedFileEntryCache.verify()
+  // restore mocked objects
+  mockedFileEntryCache.restore()
```

Comment:
agree, line 23 doesn't make sense. Line 24 restores file-entry-cache dependency for further use in case of new tests.